website/docs/r/gcp_auth_backend.html.md

```diff
@@ -30,6 +30,21 @@ The following arguments are supported:
 
 * `local` - (Optional) Specifies if the auth method is local only.
 
+* `custom_endpoint` - (Optional) Specifies overrides to
```

Comment:
Maybe this is a good candidate for an example TF config in the docs?

Reply:
Definitely.

Reply:
Probably want to document that this is a `1.11+` feature.

Reply:
Updated the example TF config in the docs to include the new param. I noted that 1.11+ is required. Let me know if that looks okay. [747f2b5](https://github.com/hashicorp/terraform-provider-vault/pull/1482/commits/747f2b5890b7843c98a70822447f99f1e29a5e34)